src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -146,7 +146,7 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
           uri: dataToUse.uri ? dataToUse.uri : '',
           activity_id: dataToUse.activity_id,
           project_id: dataToUse.project_id,
-          start_date: new Date(),
+          start_date: new Date(new Date().setSeconds(0)),
```

Comment:
The same suggestion as above.

Reply:
Thank very much for your feedback @jcalarcon98, I really appreciate that. I have implemented your suggestions, but with some other minor changes I considered necessary.